README.md

```diff
@@ -1,2 +1,32 @@
 # conferences
 Slides, white papers and agendas from various events in which 3mdeb participated
+
```

Comment:
@artur-rs looks good so far, need the links now to the conference READMEs.

Reply:
@miczyg1 cannot do better, that is right now